sys/include/net/gnrc/netreg.h

```diff
@@ -212,6 +218,11 @@ typedef struct gnrc_netreg_entry {
 void gnrc_netreg_init(void);
 
 /**
+ * @name    Dynamic entry initialization functions
```

Comment:
Because they are macros... This

```C
void some_name(int param);
```

is a function definition. This

```C
#define some_name(param)    (param + 1)
```

is a macro declaration ;-)